lib/voyager-message/src/module.rs

```diff
@@ -176,12 +176,18 @@ impl ClientModuleInfo {
 
     pub fn ensure_ibc_interface(
         &self,
-        ibc_interface: impl AsRef<str>,
+        expected_interfaces: Vec<impl AsRef<str>>,
     ) -> Result<(), UnexpectedIbcInterfaceError> {
-        if ibc_interface.as_ref() != self.ibc_interface.as_str() {
+        if !expected_interfaces
+            .iter()
+            .any(|expected| expected.as_ref() == self.ibc_interface.as_str())
+        {
             Err(UnexpectedIbcInterfaceError {
                 expected: self.ibc_interface.clone(),
-                found: ibc_interface.as_ref().to_owned(),
+                found: expected_interfaces
+                    .iter()
+                    .map(|s| s.as_ref().to_string())
+                    .collect(),
```

Comment:
i think we actually have all of these backwards now that i look at it - all of the `Unexpected*Error` `found`/`expected` semantics should be switched such that the `found` contains what was in the config (i.e. `self.ibc_interface` in this context) and expected contains the list of supported values.

Reply:
gotcha, will fix.